src/fluree/http_api/handlers/ledger.clj

```diff
@@ -117,35 +117,28 @@
 
 (def query
   (error-catching-handler
-    (fn [{:keys [fluree/conn content-type credential/did] {{:keys [ledger query] :as body} :body} :parameters}]
-      (log/debug "query handler received body:" body)
+    (fn [{:keys [fluree/conn content-type credential/did] {{ledger :from :as query} :body} :parameters}]
+      (log/debug "query handler received query:" query)
       (let [db     (->> ledger (fluree/load conn) deref! fluree/db)
-            opts (cond-> (query-body->opts body content-type)
+            opts (cond-> (query-body->opts query content-type)
```

Comment:
Really happy to see this PR!

One note, and maybe for a separate ticket. There is no longer a need to try and set `:context-type`... so we don't need the `query-body->opts` fn, nor the corresponding one for transactions.